File: public/phase-2-20160620022203/JAVA/eshopping.java
// Name : EShopping
import java.util.*;
import java.lang.*;
import java.io.*;
class Main {
	public static void main (String[] args) throws java.lang.Exception {
		// input:
		// n :  number of phone models' descriptions in the shop
		// description :  zero stands for an unread description one for a read description
		int index;
		Scanner in = new Scanner(System.in);
		int n = Integer.parseInt(in.nextLine());
		int[] description = new int[n];
		String[] description_elements = (in.nextLine()).split(" ");
		for(index=0;index<n;index++)
			description[index] = Integer.parseInt(description_elements[index]);


		// write your code here
		// store your results in `clicks`

		// output
		// Dummy Data
		int clicks=9;

		System.out.println(clicks);

	}
}

File: public/phase-2-20160620022203/C/eshopping.c
// Name : EShopping
#include <stdio.h>
int main() {
	// input:
	// n :  number of phone models' descriptions in the shop
	// description :  zero stands for an unread description one for a read description
	int n, description[1000], clicks=9, index;
	scanf("%d", &n);
	for(index = 0; index< n; index++) {
		scanf("%d", &description[index]);
	}


	// write your code here
	// store your results in `clicks`

	// output
	printf("%d", clicks);
	return 0;
}